src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -377,10 +383,10 @@ public String actualPathForPublicPath(String publicPath) {
         if (publicPath.contains("files/temp")) {
             return Path.of(FilePathService.getTempFilePath(), filename).toString();
         }
-        if (publicPath.contains("files/drag-and-drop/backgrounds")) {
+        if (publicPath.contains(FileService.DRAG_AND_DROP_BACKGROUND_SUBPATH)) {
```

Comment:
optional (just resolve if you don't like it :) )
```suggestion
        if (publicPath.contains(DRAG_AND_DROP_BACKGROUND_SUBPATH)) {
```